<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>518</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>585</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icone.ico</normaloff>icone.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0.486, y1:0, x2:0.497, y2:1, stop:0 rgba(42, 71, 94, 255), stop:1 rgba(28, 41, 56, 255))</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0.557, x2:1, y2:0.545, stop:0 rgba(6, 191, 255, 255), stop:1 rgba(45, 115, 255, 255));
border-radius:85px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>187</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>23</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255,255,255);
background-color:transparent
</string>
        </property>
        <property name="text">
         <string>BEM-VINDO</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>imgs/pngegg.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="txt_username">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgb(255, 255, 255);
border-radius:10px;
padding:5px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>USERNAME</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="txt_password">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgb(255, 255, 255);
border-radius:10px;
padding:5px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>SENHA</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="botton_login">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  color: black;
  background-color: rgb(255,255,255);
border-radius:10px
}
QPushButton:hover{
 
  background-color:rgb(207, 207, 207)

}
QPushButton:pressed {
background-color: rgb(78, 125, 195);
  
}</string>
        </property>
        <property name="text">
         <string>LOGIN</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
